cli/src/main/java/org/aya/cli/parse/AyaGKProducer.java

```diff
@@ -792,6 +800,17 @@ private Arg<Pattern> unitPattern(@NotNull GenericNode<?> node) {
     return Expr.Array.newList(entireSourcePos, exprs);
   }
 
+  public @NotNull Expr.Let.Bind letBind(@NotNull GenericNode<?> node) {
+    var pos = sourcePosOf(node);
+    var bind = weakId(node.child(WEAK_ID));
+    // make IDEA happy
+    var teles = lambdaTelescope((SeqView<GenericNode<?>>) node.childrenOfType(LAMBDA_TELE));
```

Comment:
`node.childrenOfType(...).map(x -> x)`

Reply:
My stupid brain rejected this code before. 😭

Reply:
My stupid brain said the `.map(x -> x)` is inefficient.

Reply:
It is but I don't think it matters that much

Reply:
> It is but I don't think it matters that much

Yes, my bad habit.

Reply:
I think `.map` is better as it only causes compile errors, while type cast may throw runtime exceptions.

Reply:
> I think `.map` is better as it only causes compile errors, while type cast may throw runtime exceptions.

YES! I got it.